File: SDKs/DOTNet/Visual Studio/C#/GameLogic.cs
using System;
using System.Drawing;
using System.Collections.Generic;
using System.Windows.Forms;
using System.Text;

using SwinGame;
using Graphics = SwinGame.Graphics;
using Bitmap = SwinGame.Bitmap;
using Font = SwinGame.Font;
using FontStyle = SwinGame.FontStyle;
using Event = SwinGame.Event;
using CollisionSide = SwinGame.CollisionSide;
using Sprite = SwinGame.Sprite;

using GameResources;

namespace GameProject
{
    public static class GameLogic
    {
        public static void RunGame()
        {
            //Open a new Graphics Window
            Core.OpenGraphicsWindow("Game", 800, 600);
            //Open Audio Device
            Audio.OpenAudio();
            //Load Resources
            Resources.LoadResources();

            //Game Loop
            do
            {
                //Clears the Screen to Black
                Graphics.ClearScreen();

                //Hello World
                Text.DrawText("Hello World", Color.White, Resources.GameFont("Courier"), 10, 10);

                //Refreshes the Screen and Processes Input Events
                Core.RefreshScreen();
                Core.ProcessEvents();

            } while (!Core.WindowCloseRequested());

            //Free Resources and Close Audio, to end the program.
            Resources.FreeResources();
            Audio.CloseAudio();
        }
    }
}

File: SDKs/DOTNet/Visual Studio/VB/GameLogic.vb
Module GameLogic
    Public Sub Main()
        'Opens a new Graphics Window
        SwinGame.Core.OpenGraphicsWindow("Game", 800, 600)

        'Open Audio Device
        SwinGame.Audio.OpenAudio()

        'Load Resources
        Dim Resources As GameResources
        Resources = New GameResources
        Resources.LoadResources()

        'Game Loop
        Do
            'Clears the Screen to Black
            SwinGame.Graphics.ClearScreen()

            'Hello World
            SwinGame.Text.DrawText("Hello World", Color.White, Resources.GameFont("Courier"), 10, 10)

            'Refreshes the Screen and Processes Input Events
            SwinGame.Core.RefreshScreen()
            SwinGame.Core.ProcessEvents()

        Loop Until SwinGame.Core.WindowCloseRequested() = True

        'Free Resources and Close Audio, to end the program.
        Resources.FreeResources()
        SwinGame.Audio.CloseAudio()

    End Sub

End Module
